tests/conda_token/test_token_cli.py

```diff
@@ -1,13 +1,12 @@
 from datetime import datetime
 from uuid import UUID
-from uuid import uuid4
 
-import jwt
 import pytest
 from pytest_mock import MockerFixture
 from requests_mock import Mocker as RequestMocker
 
 from ..conftest import CLIInvoker
+from ..conftest import is_conda_installed
 
 pytest.importorskip("conda")
```

Comment:
This should already handle the `is_conda_installed()` function, so we don't need to import and use that in this module.